packages/devtools-network-console/src/ui/Authorization/BasicAuthorization.tsx

```diff
@@ -22,6 +22,12 @@ export interface IBasicAuthorizationProps {
 
 function BasicAuthorization(props: IBasicAuthorizationProps & ILocalized) {
     const dispatch = useDispatch();
+    React.useEffect(() => {
+        const showPasswordInput = document.querySelector('#basicAuthorizationShowPassword');
```

Comment:
Consider adding a comment explaining why this is necessary (FAST-React `Toggle` doesn't construct the label correctly).